src/gui/tray/TalkReplyTextField.qml

```diff
@@ -0,0 +1,81 @@
+import QtQuick 2.15
+import Style 1.0
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import com.nextcloud.desktopclient 1.0
+
+Item {
+    id: root
+
+    function sendReplyMessage() {
+        if (replyMessageTextField.text === "") {
+            replyMessageTextFieldBorder.border.color = Style.errorBoxBorderColor;
+            return;
+        }
+
+        replyMessageTextFieldBorder.border.color = Style.ncBlue;
+
+        UserModel.currentUser.sendReplyMessage(model.conversationToken, replyMessageTextField.text, model.messageId);
+
+        replyMessageSent.text = replyMessageTextField.text;
+        replyMessageTextField.clear();
+        replyMessageTextField.visible = false
+    }
+
+    Text {
+        id: replyMessageSent
+        font.pixelSize: Style.topLinePixelSize
+        color: Style.menuBorder
+        visible: replyMessageSent.text !== ""
+    }
+
+    TextField {
+        id: replyMessageTextField
+
+        // TODO use Layout to manage width/height. The Layout.minimunWidth does not apply to the width set.
+        height: 38
+        width: 250
+        Layout.fillWidth: true
```

Comment:
The parent of this TextField is not a Layout so this property isn't doing anything